sql-statements/sql-statement-show-table-regions.md

```diff
@@ -50,6 +50,8 @@ Executing `SHOW TABLE REGIONS` returns the following columns:
 * `READ_BYTES`: The estimated amount of data read from the Region within one heartbeat cycle. The unit is byte.
 * `APPROXIMATE_SIZE(MB)`: The estimated amount of data in the Region. The unit is megabytes (MB).
 * `APPROXIMATE_KEYS`: The estimated number of Keys in the Region.
+* `SCHEDULING_CONSTRAINTS`：The [placement rule](/placement-rules-in-sql.md) of the region which belongs to a table or partition with placement setting.
```

Comment:
We should call it `placement policy`. Only use `placement rules` if we are talking the feature name.